pkg/registry/images.go

```diff
@@ -73,6 +73,15 @@ func DeleteUnusedImages(appID string, ignoreRollback bool) error {
 		return nil
 	}
 
+	kurlRegistryHost, _, _, err := kotsutil.GetKurlRegistryCreds()
+	if err != nil {
+		return errors.Wrap(err, "failed to get kurl registry creds")
+	}
+
+	if kurlRegistryHost != registrySettings.Hostname {
+		return errors.Wrap(err, "failed to delete unused images. deleting images from an external registry is not supported")
```

Comment:
Can I know the reason behind it?

Reply:
It's because the image garbage collection code gets triggered automatically when a new version is deployed, and the error is not actionable by the user.